src/Pages/Applications/DevtronApps/Details/AppConfigurations/AppConfiguration.provider.tsx

```diff
@@ -44,9 +44,10 @@ export const AppConfigurationProvider = (props: AppConfigurationProviderProps) =
 
     const [showRepoOnDelete, setShowRepoOnDelete] = useState('')
 
-    const contextValue = useMemo(
+    const contextValue = useMemo<AppConfigurationContextType>(
         () => ({
             appId,
+            appName: state.appName,
```

Comment:
Shouldn't we read this from the detail provider?

Reply:
not clear

Reply:
There is a provider at the app detail level as well, which should have the app name and app id. Confirm with Abhishek since he has some context around the same